header-tips/index.md

```diff
@@ -0,0 +1,19 @@
+## Quick tips for making a header
```

Comment:
Let's combine this with "What images can I use for a header" Doesn't make sense for it to have it's own section.